server/routes/infoPage.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  const dummydb = require("../lib/dummydb.js");
+  const str1 = `Phonebook has info for ${dummydb.getLength()} people`;
```

Comment:
you can make this part look nicer:
```suggestion
  const str1 = `Phonebook has info for ${dummydb.getLength()} people\n${Date()}`;
```